### PR TITLE
Event reminder: Allow to send description for lectures

### PR DESCRIPTION
**Pull request by bpedersen2**
As lecture do not have a time table, the description often
is a good  substitute for reminder emails.

Change the logic to use the description for lectures if agenda
is requested.

### CODE REVIEW

**Comment by bpedersen2** [comment]
This is a mininal invasive version. The alternative would be to add a further field to the dialog and model to include the descritption as well. Then one could also use it on other event types in addtion to the agenda.

File: indico/modules/events/reminders/models/reminders.py [hunk 1]
```diff
@@ -96,6 +96,12 @@ class EventReminder(db.Model):
         nullable=False,
         default=False
     )
+    #: If the notofication should include the events description.
```
Comment (mic4ael) [inline indico/modules/events/reminders/models/reminders.py]: notification

File: indico/modules/events/reminders/forms.py [hunk 2]
```diff
@@ -56,6 +56,9 @@ class ReminderForm(IndicoForm):
     message = TextAreaField(_('Note'), description=_('A custom message to include in the email.'))
     include_summary = BooleanField(_('Include agenda'),
                                    description=_("Includes a simple text version of the event's agenda in the email."))
+    include_description = BooleanField(_('Include description'),
+                                   description=_("Includes a simple text version of the event's"
```
Comment (mic4ael) [inline indico/modules/events/reminders/forms.py]: not properly aligned

File: indico/migrations/versions/20171117_1035_40d4569835b8_add_include_description_to_reminders.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('reminders', sa.Column('include_description', sa.Boolean(), nullable=False, default=False,
+	 server_default='false'), schema='events')
+    op.alter_column('reminders', sa.Column('include_description', sa.Boolean(), nullable=False, default=False),
+	 schema='events')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```
Comment (mic4ael) [inline indico/migrations/versions/20171117_1035_40d4569835b8_add_include_description_to_reminders.py]: to remove

**Comment by ThiefMaster** [comment]
While the idea is good, I see one problem: Event descriptions can contain HTML, and the reminder emails are plain text...

BTW, 2.0 is in feature freeze; any non-bugfix contributions  should be against the v2.1-dev branch.

**Comment by bpedersen2** [comment]
New pull request against correct branch: https://github.com/indico/indico/pull/3157

**Comment by ThiefMaster** [comment]
Just FYI, you can change the upstream branch of a pull request (using the "edit" button on top, next to the title)

**Pull request closed without merging**